app/controllers/api/v1/metadata_schemas_controller.rb

```diff
@@ -0,0 +1,125 @@
+module Api
+  module V1
+    class MetadataSchemasController < ApiBaseController
+      include Swagger::Blocks
```

Comment:
since these actions all only apply to a single route, I could see either moving each before_action declaration to right before each route function, or not using before_action and just calling the methods explicitly as the first statement of each route.  Totally a matter of taste, so feel free to ignore :)

Reply:
Yeah I went back and forth about where to put these methods, and whether they should be static or not.  I'll ponder it some more and see if something else makes more sense.

Reply:
03f64f7